tests/manage/z_cluster/nodes/test_simultaneous_drain_of_two_ocs_nodes.py

```diff
@@ -0,0 +1,141 @@
+import logging
```

Comment:
I suggest we have all tests related to node drain to be under the same test file (test_node_maintenance.py)

Reply:
Done